dotfiler.sh

```
@@ -484,7 +511,9 @@ while [ "$operations" != "" ]; do
 	while [ "$local_systems" != "" ]; do
 		current_sys="$(Front "$local_systems")"
 		local_systems="$(Pop "$local_systems")"
-
+		current_systag="-$current_sys"
+		( [ -f "$conf_dir/filelist$current_systag" ] ) 2>/dev/null || CreateFileList $current_systag #Prompt if missing config
+		[ -d "$conf_dir/files$current_systag/" ] || mkdir -p "$conf_dir/files$current_systag" #Create files directory
```

Since this operation is only relevant to `Pull`, it should me moved to that function. The comments should be removed and the files directory is created there, so the `mkdir -p` line should also be removed. This operation is not relevant to `Deploy` because creating a missing files directory there is a futile operation, so it should just return an error. I've also noticed that files are being pulled to `$conf_dir` here and in many other lines. Is relocating the pulled dotfiles' location the purpose of this PR? I had understood different and don't know what you're trying to do here.